omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -1,16 +1,37 @@
 package org.openmrs.module.locationbasedaccess.fragment.controller.field;
 
+import org.apache.commons.lang3.StringUtils;
 import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.PersonAttribute;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.module.locationbasedaccess.PatientSearchAdviser;
 import org.openmrs.ui.framework.fragment.FragmentModel;
 import org.openmrs.api.context.Context;
+import org.springframework.web.bind.annotation.RequestParam;
+
 import javax.servlet.http.HttpSession;
 import java.util.List;
 
 public class LocationsFragmentController {
 
     public void controller(FragmentModel model,
-                           HttpSession session) {
+                           HttpSession session,
+                           @RequestParam(value = "patientId", required = false) Patient patient) {
         List<Location> activeLocations = Context.getLocationService().getAllLocations();
         model.addAttribute("activeLocations", activeLocations);
+
+        model.addAttribute("selectedLocationUuid", null);
+        if (patient instanceof Patient) {
```

Comment:
What is the use of the above line?

Reply:
When we go to the Patient Registration, patintId will be null. So the patient will be also null. 
When we go to the Location Editing, patintId will exist and patient also. So it will be used to track that two usages as well.

Reply:
When you use the instanceof operator, the message you send across is that there is a possibility of having an instance which is not of type Patient.

Reply:
Yes, Got it. 
I have changed it to null checking.